src/internal.c

```diff
@@ -3762,10 +3763,12 @@ void InitX509(WOLFSSL_X509* x509, int dynamicFlag, void* heap)
     InitX509Name(&x509->issuer, 0, heap);
     InitX509Name(&x509->subject, 0, heap);
     x509->dynamicMemory  = (byte)dynamicFlag;
-    #if defined(OPENSSL_EXTRA_X509_SMALL) || defined(OPENSSL_EXTRA)
+    #if defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL)
```

Comment:
So this section is not available for `OPENSSL_EXTRA_X509_SMALL` now? Is that on purpose?